images/origin/system-container/config.json.template

```diff
@@ -70,15 +70,13 @@
 	    "source": "proc"
 	},
 	{
-	    "destination": "/dev",
-	    "type": "tmpfs",
-	    "source": "tmpfs",
-	    "options": [
-		"nosuid",
-		"strictatime",
-		"mode=755",
-		"size=65536k"
-	    ]
+		"destination": "/dev",
+		"type": "bind",
+		"source": "/dev",
+		"options": [
+			"rbind",
+			"rprivate"
```

Comment:
`rslave` works fine on our nodes. Both local volumes and EBS volumes are formatted and bind mounted in the pods.

However, I'm seeing some selinux entries in our log:
`kernel: type=1401 audit(1502308394.654:179): op=security_bounded_transition seresult=denied oldcontext=system_u:system_r:container_runtime_t:s0 newcontext=system_u:system_r:mount_t:s0` when the volumes are attached to pods. Strange, because everything seems to work anyway.

Should I update the PR to use `rslave`instead @giuseppe ?